api/python/provisioner/__main__.py

```diff
@@ -132,6 +132,9 @@ def _set_logging(output_type, log_args=None, other_args=None):
         setattr(log_args, config.LOG_RSYSLOG_HANDLER, False)
         log_args.update_handlers()
 
+    if cmd in config.LOG_SUPPRESSION_CMDS:
+        setattr(log_args, config.LogLevelTypes.RSYSLOG.value, logging.WARNING)
```

Comment:
please add a conditional `if hasattr(log_args, config.LOG_RSYSLOG_HANDLER):` for cases when rsyslog is not listed in handlers

Reply:
Done.
But as improvement we need to consider a way to combine `hasattr` and `setattr` method in single call. For example, `attr` module support `on_setattr` callback
https://www.attrs.org/en/stable/api.html

Reply:
sure, you can do the poc here